src/headline_grabber/__main__.py

```diff
@@ -30,7 +30,16 @@
     callback=validate_site_name,
     help="Comma-separated list of news sources to exclude from the search",
 )
-def main(include: str, exclude: str):
+@click.option(
+    "--entries",
```

Comment:
Not a fan of the option naming. In SQL there's `LIMIT` which puts a cap on the number of results the SQL query returns. Maybe let's riff off that and call the option `--limit` or something similar. Maybe even clearer would be `--section-limit` or `--section-max`

Reply:
I named it limit and used -l.